### PR TITLE
Add a comment on why we disabled Direct HTTP streams

### PR DESCRIPTION
**Pull request by Shredder121**
Do you know what markdown syntax we use?
And if strikethrough is supported?

Or do you want a much bigger notice, pulled apart from the features?

### CODE REVIEW

File: public/markdown/index.md [hunk 1]
```diff
@@ -59,10 +59,12 @@ You can then choose your song with the `;;select <number>` command.
 
 * Vimeo
 
-* Direct links to files, including streams. Not all formats supported.
+* ~~Direct links to files, including streams. Not all formats supported.~~¹
```
Comment (Frederikam) [inline public/markdown/index.md]: The `~~` syntax only works in Discord AFAIK

**Comment by Shredder121** [comment]
http://c1e82577.ngrok.io/ running on my local machine

File: public/markdown/index.md (hunk 1 above)
Comment (Shredder121) [inline public/markdown/index.md]: It does work when I look at it though.

**Comment by Frederikam** [comment]
image

This sounds plural as if multiple features are disabled. Be specific here

**Comment by Shredder121** [comment]
Oh, I asked if we also blocked arbitrary `.m3u` or `.pls` files.
So I had applied strikethrough with both of them, so that's why "these".
I _think_ those lists are now also blocked, right?

**Comment by Frederikam** [comment]
Yes they are, as those are served over HTTP(S)

**Comment by Shredder121** [comment]
Is it better now? or should I change `these`=>_something else_
Files served over HTTP?
Direct links?

**Comment by Frederikam** [comment]
>Files served over HTTP

Sounds good

**Pull request merged**